packages/verifiable-credentials/test/config.ts

```diff
@@ -0,0 +1,58 @@
+import { AutoAccount } from '@verida/account-node';
+import { Context, EnvironmentType, Network } from '@verida/client-ts';
+
+export const config = {
+    PRIVATE_KEY_1: "0x5dd84b6d500bcbc018cbc71b0407d694095755d91af42bd3442b2dfc96b1e0af",
+    PRIVATE_KEY_2: "0x80d3b996ec98a91536efdffbae40f5eaaf117765a587483c69195c9460165c37",
+    ISSUER_DID: 'did:vda:0xB3729982A2585544FD72c99CF3773a9c6baBD55c',
+    SUBJECT_DID: 'did:vda:0xC2629982A2585544FD72c99CF3773a9c6baBD55c',
+    ADDRESS_1: '0xB3729982A2585544FD72c99CF3773a9c6baBD55c',
+    CREDENTIAL_DATA: {
+        name: 'Vitalik Buterin',
+        firstName: 'Vitalik',
+        lastName: 'Buterin',
+        email: 'me@vitalik.eth',
+        schema: 'https://common.schemas.verida.io/social/contact/v0.1.0/schema.json'
+    },
+    INVALID_CREDENTIAL_DATA: {
+        email: 'me',
+        schema: 'https://common.schemas.verida.io/social/contact/v0.1.0/schema.json'
+    },
+    VERIDA_CONTEXT_NAME: "Verida: Credentials",
+    VERIDA_EXPECTED_DATABASE: 'credential_public_encrypted',
+    VERIDA_TESTNET_DEFAULT_SERVER: "https://db.testnet.verida.io:5002/",
+    VALID_VERIDA_URI: `verida://did:vda:0xB3729982A2585544FD72c99CF3773a9c6baBD55c/0x42406d644c8e884be0f76ee05539cfba03eda65baa2bda33ecd364ce4d18047c/credential_public_encrypted/4f54db10-8b92-11ec-b09c-bf8dd364cfc7?key=9dc07aae40b856cecb7c257b73c7929532284f5be1c153916da754cf841bdb69`,
```

Comment:
@cmcWebCode40  Are these URI's and INVALID_VC_JWT still required?

Reply:
I have removed all expect for INVALID_VERIDA_URI used to test `When Verida URI data does not exist in the database`